scripts/core/queue.yml

```diff
@@ -46,6 +46,18 @@ etors:
       value: "$X_BIT(3)"
       desc: "Is the default queue for a device"
 --- #--------------------------------------------------------------------------
+type: enum
+desc: "Queue Properties"
+class: $xQueue
+name: $x_queue_props_t
```

Comment:
Ideally this would be called `$x_queue_properties_x` however the code generation will match this as a properties struct and not an enum and fail to generate the spec [here](https://github.com/oneapi-src/unified-runtime/blob/main/scripts/parse_specs.py#L691) which matches anything ending in `properties_t`.